LICENSE.md

```diff
@@ -0,0 +1,14 @@
+[BSD 3-Clause License](https://choosealicense.com/licenses/bsd-3-clause/)
+
+Copyright (c) 2007-2020, Till Brehm, ISPConfig UG
```

Comment:
These plugins are not maintained by the ISPConfig developers, so no copyright for us.